### ISSUE TITLE
UnitPriceExclVAT in CZL Reports shows invalid prices

### ISSUE THREAD

**Issue by deerware** [microsoft/ALAppExtensions#15420]
"Sales Cr.Memo Line"."Unit Price" sometimes already has a Unit Price without VAT, if I'm not mistaken it depends on the "Prices Including VAT" property in the header.
This results in VAT being substracted from a price that is already without VAT, which them shows invalid numbers.

https://github.com/microsoft/ALAppExtensions/blob/d35a1425c93e0a72c17e3246a5466c91be1ccd9c/Apps/CZ/CoreLocalizationPack/app/Src/Reports/SalesCreditMemo.Report.al#L369

Here's some sample from SalesLine.Table.al where it works correctly
![image](https://user-images.githubusercontent.com/7270284/144230947-84e19429-06b7-4967-9a85-0ae858e4a043.png)

**Comment by DavidHolusa** [comment]
@deerware What version of bc are we talking about here? In the current version of bc17, 18 and 19 is the following code:
```
trigger OnAfterGetRecord()
begin
    UnitPriceExclVAT := "Unit Price";
    if "Sales Cr.Memo Header"."Prices Including VAT" then
        UnitPriceExclVAT := Round("Unit Price" / (1 + "VAT %" / 100), Currency."Amount Rounding Precision");
end;
```
and this code should be work correctly.

**Comment by deerware** [comment]
@DavidHolusa It was in BC19, but it has been fixed in commit 95ddcf4 "19.2 changes".
I've marked the original line in my original post, but you can consider this issue resolved.

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.
